bot/exts/utils/makegurkan.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext import commands
+from discord.utils import find
+import re
+
+
+class MakeGurkan(commands.Cog):
+    def __init__(self, bot):
```

Comment:
```suggestion
    def __init__(self, bot: Bot) -> None:
```
Just remember to import it:
```py
from bot.bot import Bot
```